test/chrome/chromeDebugAdapter.test.ts

```diff
@@ -129,8 +129,11 @@ suite('ChromeDebugAdapter', () => {
 
     suite('attach()', () => {
         test('Initialized event is fired after first scriptParsed event', done => {
```

Comment:
I'd update this to change the new test:
Initialized event and script events are fired after first scriptParsed event